packages/wonder-blocks-banner/src/components/__tests__/banner.test.js

```diff
@@ -340,4 +340,25 @@ describe("Banner", () => {
         const banner = screen.getByTestId("wonder-blocks-banner-test-id");
         expect(banner).toHaveAttribute("aria-live", "polite");
     });
+
+    test("ariaLabel prop becomes aria-label on the banner", () => {
+        // Arrange
+
+        // Act
+        render(
+            <Banner
+                text=""
```

Comment:
**question:** why is `text` empty?

Reply:
oh it doesn't have to be. I can add something. I figured the text wasn't important since this is inside a unit test.